app/views/shared/_dashboard_navbar.html.erb

```diff
@@ -9,7 +9,6 @@
           <li class="nav-item">
             <%= link_to boards_path, class: "nav-link px-0" do %>
```

Comment:
```suggestion
            <%= link_to "Boards", boards_path, class: "nav-link px-0 #{'active' if action_name == 'boards'}" %>
```

Reply:
Maybe you can try using if statement on the `link_to` directly so we can remove the stimulus controller :D

Reply:
The same thing can be applied for the Feedback `link_to` 😄